### ISSUE TITLE
Create a keyboard shortcut for 'Connect Clip at Playhead'

### ISSUE THREAD

**Issue by randomeizer** [fcpxhacks/fcpxhacks#84]
Currently, it is only possible to change the connection point of a clip by pressing Cmd+Opt then clicking the desired connection point. This is quite imprecise, and in particular it's really difficult to move the connection point to the absolute beginning or end of a clip this way.

It would be great if we could create a custom command that would move the connection point for the currently-selected clip to the playhead (assuming they intersect). I imagine the procedure would be something along the lines of:

1. Figure out what the currently selected clip is.
2. Figure out where the playhead is.
3. Move the mouse to where the selected clip and playhead intersect.
4. Send a Cmd+Opt+Click
5. Return the mouse to it's original location.

Possible?

**Comment by latenitefilms** [comment]
Yes, this is definitely possible. Anything where you have to programatically control the mouse to do things is always a bit messy - but definitely worth a shot!
